src/base_client.rs

```diff
@@ -265,6 +205,44 @@ impl Client {
             ))))
     }
 
+    /// Handle a m.ignored_user_list event, updating the room state if necessary.
+    ///
+    /// Returns true if the room name changed, false otherwise.
+    pub(crate) fn handle_ignored_users(&mut self, event: &IgnoredUserListEvent) -> bool {
+        // TODO use actual UserId instead of string?
```

Comment:
This is a bit of an annoying issue, UserId should really be just a wrapper around String and we should be able to convert easily from one to the other and from `UserId` to `&str` but that needs to be changed in ruma-identifiers.

So I guess leave it as string for now.